src/Discord.Net.WebSocket/Entities/Interaction/SocketInteraction.cs

```diff
@@ -96,11 +96,12 @@ internal virtual void Update(Model model)
         ///     Responds to an Interaction with type <see cref="InteractionResponseType.ChannelMessageWithSource"/>.
         /// <para>
         ///     If you have <see cref="DiscordSocketConfig.AlwaysAcknowledgeInteractions"/> set to <see langword="true"/>, You should use
-        ///     <see cref="FollowupAsync(Discord.Embed[],string,bool,bool,Discord.AllowedMentions,Discord.RequestOptions,Discord.MessageComponent)"/> instead.
+        ///     <see cref="FollowupAsync"/> instead.
         /// </para>
         /// </summary>
         /// <param name="text">The text of the message to be sent.</param>
         /// <param name="embeds">A array of embeds to send with this response. Max 10</param>
+        /// <param name="embed">DEPRECTAED single embed to send in followup</param>
```

Comment:
move the `embed` param tag to the end of the param list and remove `DEPRICATED` as its not anymore